rhubarb/CMakeLists.txt

```diff
@@ -276,6 +276,40 @@ target_link_libraries(vorbis
 target_compile_options(vorbis PRIVATE ${disableWarningsFlags})
 set_target_properties(vorbis PROPERTIES FOLDER lib)
 
+# ... opus
+FetchContent_Declare(opus
+	GIT_REPOSITORY https://github.com/xiph/opus.git
+        GIT_TAG v1.5.2
+)
+set(OPUS_BUILD_SHARED_LIBRARY OFF)
+set(OPUS_BUILD_TESTING OFF)
+set(OPUS_BUILD_PROGRAMS OFF)
+set(OPUS_CUSTOM_MODES ON)
+message(STATUS "Fetching opus library...")
+FetchContent_MakeAvailable(opus)
+message(STATUS "Fetching opus library... done.")
+include_directories(SYSTEM "${CMAKE_BINARY_DIR}/_deps/opus-src/include")
```

Comment:
That's not a reliable way of referencing the lib code. Better use `opus_SOURCE_DIR`.